docs/src/pages/components/data-grid/virtualization/ColumnVirtualizationGrid.tsx

```diff
@@ -45,11 +45,11 @@ function useData(rowLength: number, columnLength: number) {
 }
 
 export default function ColumnVirtualizationGrid() {
-  const data = useData(50, 1000);
+  const data = useData(500, 1000);
 
   return (
     <div style={{ height: 400, width: '100%' }}>
-      <DataGrid {...data} columnBuffer={2} />
+      <XGrid {...data} columnBuffer={2} hideFooter />
```

Comment:
This is not a XGrid only feature and the header doesn't have the pro plan indicator. It's not consistent.

I propose we move in this direction to be more competitive with ag-Grid:

```suggestion
      <DataGrid {...data} columnBuffer={2} hideFooter />
```

Reply:
I would personally remove 
```suggestion
      <XGrid {...data} columnBuffer={2} />
```

To keep the inline preview as clear as possible, 

<img width="846" alt="Screenshot 2021-04-08 at 12 01 54" src="https://user-images.githubusercontent.com/3165635/114008131-3b0f2f80-9862-11eb-889e-6bc4e47f2df0.png">

Reply:
dataGrid only allows 100 rows, it's too short to show the virtualization power